tests/unit/igraph_wheel.c

```diff
@@ -0,0 +1,61 @@
+/* IGraph library.  Copyright (C) 2021  The igraph development team <igraph@igraph.org>
+
+   This program is free software; you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation; either version 2 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#include <igraph.h>
+#include "test_utilities.inc"
+
+void call_and_print(igraph_t *graph, igraph_integer_t n, igraph_wheel_mode_t mode,
+                igraph_integer_t center) {
+
+    IGRAPH_ASSERT(igraph_wheel(graph, n, mode, center) == IGRAPH_SUCCESS);
+    print_graph_canon(graph);
+    igraph_destroy(graph);
+    printf("\n");
+}
+
+int main() {
+    igraph_t g_1, g_2, g_3, g_4, g_5, g_6, g_7, g_8;
```

Comment:
Can you please re-use a single variable (`graph` or `g`) for all graphs?  Create one, print, the destroy. Then re-use the same variable for the next one.

Reply:
Changed to a single variable 'graph'.